concordium-contracts-common/src/traits.rs

```diff
@@ -180,7 +180,7 @@ impl Write for Vec<u8> {
 
     #[inline]
     fn write(&mut self, buf: &[u8]) -> Result<usize, Self::Err> {
-        let _ = self.extend_from_slice(buf);
+        self.extend_from_slice(buf);
```

Comment:
Why is this unrelated thing changed?

Reply:
Clippy was complaining.

Reply:
Well please use clippy that is in the CI, not a newer version. We don't want noise in PRs.